src/components/policy/policy_external/src/sql_pt_queries.cc

```diff
@@ -77,7 +77,7 @@ const std::string kCreateSchema =
     "  `vehicle_year` VARCHAR(4), "
     "  `preloaded_date` VARCHAR (10), "
     "  `certificate` VARCHAR (45), "
-    "  `user_consent_passengersRC` BOOL,"
+    "  `user_consent_passengersRC` BOOL, "
```

Comment:
same here, if a consent is needed, change the name to something else, if not needed, please remove